packages/jobs/src/types.ts

```diff
@@ -2,18 +2,22 @@
 // debug messages. RedwoodJob will fallback to use `console` if no
 // logger is passed in to RedwoodJob or any adapter. Luckily both Redwood's
 
+import type { IntRange } from 'type-fest'
```

Comment:
We're already using type-fest in other places, so might as well use it here as well